types/i2c-bus/index.d.ts

```diff
@@ -1,19 +1,30 @@
-// Type definitions for i2c-bus v1.0.0
+// Type definitions for i2c-bus v5.1.0
 // Project: https://github.com/fivdi/i2c-bus
 // Definitions by: Jason Heard <https://github.com/101100>
+//                 Kohei Watanabe <https://github.com/kou029w>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.7
```

Comment:
This line specifies typescript 3.7 as the *minimum* version. I think these types will actually work with 2.8+, which is what DT supports right now. In a follow-up PR, you might try removing this line entirely and seeing whether tests still pass.